sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -1267,6 +1284,8 @@ _pi_context::getAvailableCommandList(pi_queue Queue,
   std::tie(CommandList, std::ignore) = Queue->CommandListMap.insert(
       std::pair<ze_command_list_handle_t, pi_command_list_info_t>(
           ZeCommandList, {ZeFence, true, ZeCommandQueue, QueueGroupOrdinal}));
+  if (auto Res = Queue->insertActiveBarriers(CommandList, UseCopyEngine))
```

Comment:
Do you really need to insert them both here and at line 1175 (right after this call)?